otherlibs/unix/socketaddr.h

```diff
@@ -17,7 +17,25 @@
 #define CAML_SOCKETADDR_H
 
 #include "caml/misc.h"
-#ifndef _WIN32
+
+#ifdef _WIN32
+#ifdef HAS_AFUNIX_H
+#include <afunix.h>
+#else
+#define UNIX_PATH_MAX 108
+
+typedef struct sockaddr_un
+{
+    ADDRESS_FAMILY sun_family;     /* AF_UNIX */
+    char sun_path[UNIX_PATH_MAX];  /* pathname */
+} SOCKADDR_UN, *PSOCKADDR_UN;
+
+// Returns ULONG PID of the connected peer process
```

Comment:
To look a little less lifted straight out of an MS header...
```suggestion
typedef struct sockaddr_un
{
    ADDRESS_FAMILY sun_family;
    char sun_path[UNIX_PATH_MAX];
} SOCKADDR_UN, *PSOCKADDR_UN;

```